### DISCUSSION TITLE
SLO Checks over a time frame

### DISCUSSION THREAD

**Discussion by kavinarvind** [artilleryio/artillery#1843] [Artillery]
How to apply threshold checks for a particular period of the load test, say steady state from (15m: 45m)? Is that possible?

**Comment by hassy** [comment]
hi @kavinarvind 👋 not possible at the moment, but it's a great feature request.

For your use-case, would running the check at the end of the test be enough? Or would you need it to run continuously as the test is running (e.g. from 15m to 45m into the test run)

**Reply by kavinarvind** [reply]
Hi Hassy, Running it at the end of the test would be enough for my scenario.
